index.html

```diff
@@ -173,6 +173,57 @@ <h3>Skills</h3>
           </li>
         </ul>
       </section>
+      <footer  class="contact-form-section d-flex flex-center">
```

Comment:
- [ ] It would be better if we change `<footer>` tag to `<section>` so that we improve our HTML semantics. 
**NB**: 
 `<footer>` tag should come after the `<main>` tag

Reply:
Thanks for the review, I wanted to clarify something before stating make  changes according to the review

- [ ]  Based on the Project requirements they said to put `the contact form in the footer` that means we suppose to have a footer in our project so can you clarify me on that point, do we have to change the`<footer>` tag in `<section>` tag or we should put the `<footer>` tag which includes the contact form and the social icons out of the `<main>` tag?

Reply:
how about bringing the footer after the contact section... Here is how the structure should be 
```
<header>
<main>
<footer>

```

Reply:
Can you clarify for me, please?
As I have understood, I have to create a new `<section>` tag only for the contact form  and put it in the `<main>` tag and create a `<footer>` tag for the social icons and others after the `<main>` tag